system-addon/content-src/message-center/message-format.md

```diff
@@ -0,0 +1,13 @@
+## Snippets message format
+
+Field name | Type     | Description | Example
+---        | ---      | ---         | ---
+`id`       | `string` | A unique identifier for the snippet that should not conflict with any other previous snippet
+`template` | `string` | An id matching an existing Activity Stream Router template | [`simple_snippet`](https://github.com/piatra/activity-stream/blob/ea5f890c9a94ddd03ffca4ccc432936a2d6795aa/system-addon/lib/MessageCenterRouter.jsm#L5-L17)
```

Comment:
Personally I would add a complete example above/below instead of linking to these examples, since this schema might be changed/updated